doc/appendix-de-kassensichv/examples/pos-receipt.md

```diff
@@ -0,0 +1,468 @@
+            //0x4445000000000012 (finish-transaction-result)
+            "ftSignatureType": 4919338167972134930,
+            "caption": "finish-transaction-payload",
+            "data": "[finishTransactionResult]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000013 (qr-code-version)
+            "ftSignatureType": 4919338167972134931,
+            "caption": "qr-code-version",
+            "data": "V0"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000014 (processType)
+            "ftSignatureType": 4919338167972134933,
+            "caption": "processType",
+            "data": "Kassenbeleg-V1"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000015 (processData)
+            "ftSignatureType": 4919338167972134934,
+            "caption": "processData",
+            "data": "Beleg^3.80_9.20_0.00_0.00_0.00^13.00:Bar"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000017 (<transaktions-zähler>)
+            "ftSignatureType": 4919338167972134935,
+            "caption": "transaktions-nummer",
+            "data": "[finish-transaktions-zähler]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x4445000000000018 (<finish-signatur-zäher>)
+            "ftSignatureType": 4919338167972134936,
+            "caption": "<signatur-zaehler> ",
+            "data": "[finish-signatur-zaehler]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            // 0x4445000000000019  <start-zeit>
+            "ftSignatureType": 4919338167972134937,
+            "caption": "<start-zeit>",
+            "data": "[start-zeit]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x444500000000001A <log-time>
+            "ftSignatureType": 4919338167972134938,
+            "caption": "<log-time>",
+            "data": "[log-time]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x444500000000001B <sig-alg>
+            "ftSignatureType": 4919338167972134939,
+            "caption": "<sig-alg>",
+            "data": "[<sig-alg>]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x444500000000001C <log-time-format>
+            "ftSignatureType": 4919338167972134940,
+            "caption": "<log-time-format>",
+            "data": "[<log-time-format>]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x444500000000001D <signatur>
+            "ftSignatureType": 4919338167972134941,
+            "caption": "<signatur>",
+            "data": "[<signatur>]"
+        },
+        {
+            "ftSignatureFormat": 1,
+            //0x444500000000001E <public-key>
+            "ftSignatureType": 4919338167972134942,
+            "caption": "<public-key>",
+            "data": "[<public-key>]"
+        }
+    ],
+    "ftState": 4707387510509011000
```

Comment:
This state is for AT `‭0x4154000000000038‬` and I guess this is a typo. Since we have not defined states yet for Germany I would say we just use a default state here.

Reply:
fixed. used default state.